lib/back.js

```diff
@@ -163,7 +163,6 @@ var record = {
     recorder.clear();
     nock.cleanAll();
     nock.activate();
-    nock.disableNetConnect();
```

Comment:
I don’t know why that was here before, but this change seems unrelated to your changes?

Reply:
It's actually related, otherwise you have to manually enable it in order for the requests to go through and be seen by the recorder. 

The **record** mode should not _disableNetConnect_ by default. For that behavior the other modes can be used.